db/migrate/20230329180317_add_u_sers_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUSersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
- Hi, since you are referencing the user, it should be singular `user` and not plural `users` because when I run your migration with `rails db:migrate`, the `schema.rb` file shows the foreign keys in plural instead of being singular. See this below:

`add_foreign_key "comments", "users", column: "users_id"` it should be 
`add_reference :comments, :user, null: false, foreign_key: true` to generate the corresponding  `user_id` which should be this 
`add_foreign_key "comments", "users"` in your schema file. kindly check the erd diagram to confirm so you can fix this.

You can check out this [rails migration](https://guides.rubyonrails.org/active_record_migrations.html), pay attention to the `add_reference` call. see this image:


![Screenshot from 2023-03-30 12-17-55](https://user-images.githubusercontent.com/59397159/228820021-2d835486-8c0e-43c6-8a32-47fe07bb517e.png)